core/src/main/java/com/amplifyframework/auth/TOTPSetupDetails.kt

```diff
@@ -12,18 +12,33 @@
  * express or implied. See the License for the specific language governing
  * permissions and limitations under the License.
  */
-package com.amplifyframework
+package com.amplifyframework.auth
 
 import android.net.Uri
 
+/**
+ * Details of TOTP Setup that help launch into TOTP manager.
+ *
+ * @param sharedSecret Secret code returned by the service to help setting up TOTP
+ * @param username username that will be used to construct the URI
+ */
 data class TOTPSetupDetails(
     val sharedSecret: String,
     val username: String
 ) {
+
+    /**
+     * Returns a TOTP setup URI that can help the customers avoid barcode scanning and use native password manager
```

Comment:
Should avoid saying "the customers" in API docs, as they are already the intended consumer of the documentation

Reply:
Yeah I copied from swift. Will update